docs/antrea-l7-network-policy.md

```diff
@@ -7,7 +7,9 @@
 - [Prerequisites](#prerequisites)
 - [Usage](#usage)
   - [HTTP](#http)
-    - [More examples](#more-examples)
+    - [More examples - HTTP](#more-examples-http)
```

Comment:
This is inconsistent with the actual section name. TOC check should be failing.
And I feel redundant to append HTTP when its parent section is already HTTP. If you are trying to avoid section name conflict, the second anchor of the second section will have "-1", and toc generator tool will do it automatically.

Reply:
I didn't know that we could generate TOC by calling `make toc`. I have updated and checked.